projects/sport-data-valley/seeds/questionnaires/psychological_competencies.rb

```diff
@@ -0,0 +1,416 @@
+# frozen_string_literal: true
+
+db_title = ''
+db_name1 = 'psychological_competencies'
+dagboek1 = Questionnaire.find_by(key: File.basename(__FILE__)[0...-3])
+dagboek1 ||= Questionnaire.new(key: File.basename(__FILE__)[0...-3])
+dagboek1.name = db_name1
+
+class PsychologicalCompetenciesMethods
+  DEFAULT_QUESTION_OPTIONS = {
+    type: :likert,
+    options: [
+      { title: 'Helemaal mee oneens', numeric_value: 0 },
```

Comment:
I'm only seeing dutch text everywhere. I'm beginning to think that this questionnaire does not need translations. Correct?

Reply:
It does, but that will come in a second iteration. I talked to Dennis and what we need live for Wednesday is just the Dutch version. I'll do the English version ASAP afterwards.